products/BellHybrid/apps/application-bell-focus-timer/presenter/FocusSettingsPresenter.cpp

```diff
@@ -5,15 +5,28 @@
 #include "data/FocusCommon.hpp"
 
 #include "models/FocusSettingsModel.hpp"
+#include "models/NotificationVolumeModel.hpp"
 
 #include <ApplicationCommon.hpp>
 #include <apps-common/InternalModel.hpp>
 #include <common/widgets/list_items/Numeric.hpp>
+#include <common/widgets/list_items/NumericWithBar.hpp>
 #include <common/widgets/list_items/MinutesWithOff.hpp>
 #include <common/widgets/list_items/Multiplicity.hpp>
+#include <gui/widgets/ListViewEngine.hpp>
 
 namespace
 {
+    enum class ListIndex
+    {
+        focusTime = 0,
```

Comment:
What about capital letters for enums? ;)

Reply:
done